runtimes/protocol/getAvailableServerConfigurations.ts

```diff
@@ -0,0 +1,13 @@
+import { ProtocolRequestType } from './lsp'
+
+export const getConfigurationFromServerRequestType = new ProtocolRequestType<
+    GetConfigurationFromServerParams,
+    any[],
```

Comment:
Rather than `any[]`, can we create a interface for response? This will make it clear to anyone reading it what the response structure is going to be like.